static/form.html

```diff
@@ -1,187 +1,366 @@
-          ></textarea>
-        </div>
+            <div class="mb-6">
+                <label for="isa" class="block grey ">/* ISA */</label>
+                <select id="isa" name="isa"
+                    class="white subtle-hover bg-dark-gray mt-1 block w-full px-3 py-2 border border-zinc-400 rounded-md ">
+                    <option value="risc-iv-32" selected>risc-iv-32</option>
+                    <option value="f32a">f32a</option>
+                    <option value="acc32">acc32</option>
+                </select>
+            </div>
 
-        <div class="mb-6">
-          <label for="asm" class="block text-sm font-medium text-gray-700"
-            >Assembler Code:</label
-          >
-          <textarea
-            id="asm"
-            name="asm"
-            required
-            class="mt-1 block w-full px-3 py-2 border border-gray-300 rounded-md shadow-sm focus:outline-none focus:ring-indigo-500 focus:border-indigo-500 sm:text-sm"
-          ></textarea>
-        </div>
+            <div class="mb-6">
+                <label for="name" class="block grey">/* Author Name */</label>
+                <input type="text" id="name" name="name" required
+                    class="whitespace-nowrap overflow-x-auto subtle-hover bg-dark-gray mt-1 block w-full px-3 py-2 border border-zinc-400 rounded-md" />
+            </div>
 
-        <div class="mb-6">
-          <label for="config" class="block text-sm font-medium text-gray-700"
-            >Config (YAML):</label
-          >
-          <textarea
-            id="config"
-            name="config"
-            required
-            class="mt-1 block w-full px-3 py-2 border border-gray-300 rounded-md shadow-sm focus:outline-none focus:ring-indigo-500 focus:border-indigo-500 sm:text-sm"
-          ></textarea>
-        </div>
+            <div class="mb-6">
+                <label for="comment" class="block grey">/* Comment */</label>
+                <textarea id="comment" name="comment"
+                    class="whitespace-nowrap overflow-x-auto subtle-hover bg-dark-gray mt-1 block w-full px-3 py-2 border border-zinc-400 rounded-md"></textarea>
+            </div>
 
-        <div class="text-center">
-          <input
-            type="submit"
-            value="Submit"
-            class="w-full inline-flex justify-center py-2 px-4 border border-transparent shadow-sm text-sm font-medium rounded-md text-white bg-indigo-600 hover:bg-indigo-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500"
-          />
-        </div>
-        <div class="text-center">{{version}}</div>
-      </form>
+            <div class="mb-6">
+                <label for="asm" class="block grey">/* Assembler Code */</label>
+                <textarea id="asm" name="asm" required
+                    class="whitespace-nowrap overflow-x-auto subtle-hover bg-dark-gray mt-1 block w-full px-3 py-2 border border-zinc-400 rounded-md"></textarea>
```

Comment:
Looks like it is a nbsp problem source. If I am mistaken, whitespace-pre will resolve the problem in a better way.